utils/errors/errors.go

```diff
@@ -1,13 +1,12 @@
 package errors
 
 import (
-	"errors"
 	"fmt"
 )
 
 var (
 	// ErrConflictBATLossEvent is an error created when trying to update a bat loss event with a different amount
-	ErrConflictBATLossEvent = errors.New("unable to update bat loss events")
+	ErrConflictBATLossEvent = "unable to update bat loss events"
```

Comment:
Maybe make this back into an error, so we can use errors.Is and errors.As upstream?